backend/stand/docker-compose.yml

```diff
@@ -17,6 +17,22 @@ services:
       - postgres
       - redis
       - depot
+  taskdep:
```

Comment:
please git ignore binaries (nothing wrong with this file, i can comment only on lines of code)

Reply:
I removed all binaries